kinder/ci/workflows/super-admin-tasks.yaml

```diff
@@ -84,14 +84,13 @@ tasks:
       ${CMD} kubeadm certs check-expiration || exit 1
 
       # Delete super-admin.conf and make sure check-expiration and renew do not return errors
-      ${CMD} rm -f /etc/kubernetes/super-admin.conf
+      ${CMD} rm -f /etc/kubernetes/super-admin.conf || exit 1
       ${CMD} kubeadm certs renew super-admin.conf || exit 1
       ${CMD} kubeadm certs check-expiration || exit 1
 
       # Cleanup
-      ${CMD} rm -f /etc/kubernetes/pki/ca.*
-      ${CMD} rm -f /etc/kubernetes/pki/apiserver-kubelet-client.crt
-      ${CMD} rm -f /etc/kubernetes/*.conf
+      ${CMD} rm -f /etc/kubernetes/pki/*.* || exit 1
```

Comment:
This still does not work on my env.

```
+ docker exec kinder-super-admin-control-plane-1 rm -f '/etc/kubernetes/pki/ca.*'
```

I tried `rm -f '/etc/kubernetes/pki/*.*'` and not work as well.

Reply:
works for me. what errors are you seeing?

Reply:
```
[root@daocloud ca]# pwd
/root/ca
[root@daocloud ca]# ls
ca.crt  ca.key
[root@daocloud ca]# rm -f '/root/ca/ca.*'
[root@daocloud ca]# ls
ca.crt  ca.key
[root@daocloud ca]# rm -f '/root/ca/*.*'
[root@daocloud ca]# ls
ca.crt  ca.key
```
When will docker add `'` quota to the command?

Reply:
In https://storage.googleapis.com/kubernetes-jenkins/logs/ci-kubernetes-e2e-kubeadm-kinder-super-admin-latest/1724328029821341696/build-log.txt:

```
!MISSING! front-proxy-ca                                            
+ docker exec kinder-super-admin-control-plane-1 rm -f '/etc/kubernetes/pki/ca.*'
+ docker exec kinder-super-admin-control-plane-1 rm -f /etc/kubernetes/pki/apiserver-kubelet-client.crt
+ docker exec kinder-super-admin-control-plane-1 rm -f '/etc/kubernetes/*.conf'
```

In the yaml:

```
# Cleanup
${CMD} rm -f /etc/kubernetes/pki/ca.*
${CMD} rm -f /etc/kubernetes/pki/apiserver-kubelet-client.crt
${CMD} rm -f /etc/kubernetes/*.conf
```

Reply:
> When will docker add ' quota to the command?

no idea, it's something that `bash -c` is doing.
i need to test it.